conf/application.conf

```diff
@@ -134,3 +134,47 @@ microservice {
   }
 }
 
+api-config {
+    scopes {
+        "read:organisations-details-ho-ssp" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"]
+        }
+    }
+    endpoints {
+        corporation-tax {
+            title = "Get the organisation's Corporation Tax details"
+            endpoint = "/organisations/details/corporation-tax/?matchId=<matchId>"
+            fields {
+                A = "value/taxpayerStartDate",
```

Comment:
There is no "value" in the IF paths - have a look at the IF schemas

Reply:
Sorry @parkeradam @owen-opencast   - this is my bad. I will update the matrix and remove the value. I took one path param too many when looking at the sample responses.